Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionTabCocoa.mm

```diff
@@ -331,8 +331,8 @@
     auto *configuredExtensionController = webView.configuration.webExtensionController;
     auto *expectedExtensionController = extensionContext()->extensionController()->wrapper();
     if (!configuredExtensionController || configuredExtensionController != expectedExtensionController) {
-        RELEASE_LOG_ERROR_IF(!configuredExtensionController, Extensions, "%{public}@ returned by webViewForWebExtensionContext: is not configured with a WKWebExtensionController", webView);
-        RELEASE_LOG_ERROR_IF(configuredExtensionController && configuredExtensionController != expectedExtensionController, Extensions, "%{public}@ returned by webViewForWebExtensionContext: is not configured with the same WKWebExtensionController as extension context; %{public}@ != %{public}@", webView, configuredExtensionController, expectedExtensionController);
+        RELEASE_LOG_ERROR_IF(!configuredExtensionController, Extensions, "web view %p returned by webViewForWebExtensionContext: is not configured with a WKWebExtensionController", webView);
```

Comment:
Does a mere `%p` default to private? I would guess so. It would be nice to be explicit with our classification of the sensitivity of the data.

Reply:
will do!

Reply:
What do you mean default to private? I think %p will just log the pointer address.

Reply:
That's my understanding as well. I think as is this will log publicly, but will only log the address.

Reply:
Oh, got it. Yeah, the format specifier is different. Sounds good then!